src/Uno.UI/UI/Xaml/DependencyPropertyDetailsCollection.Bindings.cs

```diff
@@ -41,6 +42,7 @@ internal void ApplyTemplatedParent(FrameworkElement templatedParent)
 		/// </summary>
 		public void ApplyDataContext(object dataContext)
 		{
+			_bindings = new (_bindings.OrderBy(x => x, new BindingExpressionComparer()));
```

Comment:
Any performance concerns?

cc @jeromelaban @ebariche

Reply:
This is actually a mistake. I noticed the performance implications and decided to insert in the correct place inside of sorting, but I forgot to remove that line. Adding already creates a new array and copies everything over, so inserting at a certain index and appending should be about the same cost.